src/Microsoft.DotNet.GitHub.IssueLabeler/Controllers/PredictionEngineController.cs

```diff
@@ -14,18 +14,18 @@
 
 namespace Microsoft.DotNet.GitHub.IssueLabeler
 {
-    [Route("api/WebhookIssue")]
```

Comment:
I recommend keeping this route _also_ so that we can have a transition period where both old and new routes work. Then, say, after a month (or whatever), we can delete the old route. You can have as many routes as you want for a given controller, so we can recommend to anyone using the old one that they have 1 month to use the new one. That might only be me, but I will need a non-zero amount of time to make the update to the new route 😄